.dockerignore

```diff
@@ -0,0 +1,97 @@
+########################
+## Same as in .gitignore
+########################
```

Comment:
If the contents of `.dockerignore` is the same as `.gitignore`, how about using a symlink instead?

Reply:
We could do that, but how nicely does that play with windows?